stubs/opencv-python/cv2/cv2.pyi

```diff
@@ -0,0 +1,4623 @@
+Stitcher_ERR_CAMERA_PARAMS_ADJUST_FAIL: int
+Stitcher_ERR_HOMOGRAPHY_EST_FAIL: int
+Stitcher_ERR_NEED_MORE_IMGS: int
+Stitcher_OK: int
+Stitcher_PANORAMA: int
+Stitcher_SCANS: int
+Subdiv2D_NEXT_AROUND_DST: int
+Subdiv2D_NEXT_AROUND_LEFT: int
+Subdiv2D_NEXT_AROUND_ORG: int
+Subdiv2D_NEXT_AROUND_RIGHT: int
+Subdiv2D_PREV_AROUND_DST: int
+Subdiv2D_PREV_AROUND_LEFT: int
+Subdiv2D_PREV_AROUND_ORG: int
+Subdiv2D_PREV_AROUND_RIGHT: int
+Subdiv2D_PTLOC_ERROR: int
+Subdiv2D_PTLOC_INSIDE: int
+Subdiv2D_PTLOC_ON_EDGE: int
+Subdiv2D_PTLOC_OUTSIDE_RECT: int
+Subdiv2D_PTLOC_VERTEX: int
+TERM_CRITERIA_COUNT: int
+TERM_CRITERIA_EPS: int
+TERM_CRITERIA_MAX_ITER: int
+THRESH_BINARY: int
+THRESH_BINARY_INV: int
+THRESH_MASK: int
+THRESH_OTSU: int
+THRESH_TOZERO: int
+THRESH_TOZERO_INV: int
+THRESH_TRIANGLE: int
+THRESH_TRUNC: int
+TM_CCOEFF: int
+TM_CCOEFF_NORMED: int
+TM_CCORR: int
+TM_CCORR_NORMED: int
+TM_SQDIFF: int
+TM_SQDIFF_NORMED: int
+TermCriteria_COUNT: int
+TermCriteria_EPS: int
+TermCriteria_MAX_ITER: int
+UMAT_AUTO_STEP: int
+UMAT_CONTINUOUS_FLAG: int
+UMAT_DATA_ASYNC_CLEANUP: int
+UMAT_DATA_COPY_ON_MAP: int
+UMAT_DATA_DEVICE_COPY_OBSOLETE: int
+UMAT_DATA_DEVICE_MEM_MAPPED: int
+UMAT_DATA_HOST_COPY_OBSOLETE: int
+UMAT_DATA_TEMP_COPIED_UMAT: int
+UMAT_DATA_TEMP_UMAT: int
+UMAT_DATA_USER_ALLOCATED: int
+UMAT_DEPTH_MASK: int
+UMAT_MAGIC_MASK: int
+UMAT_MAGIC_VAL: int
+UMAT_SUBMATRIX_FLAG: int
+UMAT_TYPE_MASK: int
+UMatData_ASYNC_CLEANUP: int
+UMatData_COPY_ON_MAP: int
+UMatData_DEVICE_COPY_OBSOLETE: int
+UMatData_DEVICE_MEM_MAPPED: int
+UMatData_HOST_COPY_OBSOLETE: int
+UMatData_TEMP_COPIED_UMAT: int
+UMatData_TEMP_UMAT: int
+UMatData_USER_ALLOCATED: int
+UMat_AUTO_STEP: int
+UMat_CONTINUOUS_FLAG: int
+UMat_DEPTH_MASK: int
+UMat_MAGIC_MASK: int
+UMat_MAGIC_VAL: int
+UMat_SUBMATRIX_FLAG: int
+UMat_TYPE_MASK: int
+USAC_ACCURATE: int
+USAC_DEFAULT: int
+USAC_FAST: int
+USAC_FM_8PTS: int
+USAC_MAGSAC: int
+USAC_PARALLEL: int
+USAC_PROSAC: int
+USAGE_ALLOCATE_DEVICE_MEMORY: int
+USAGE_ALLOCATE_HOST_MEMORY: int
+USAGE_ALLOCATE_SHARED_MEMORY: int
+USAGE_DEFAULT: int
+VIDEOWRITER_PROP_DEPTH: int
+VIDEOWRITER_PROP_FRAMEBYTES: int
+VIDEOWRITER_PROP_HW_ACCELERATION: int
+VIDEOWRITER_PROP_HW_ACCELERATION_USE_OPENCL: int
+VIDEOWRITER_PROP_HW_DEVICE: int
+VIDEOWRITER_PROP_IS_COLOR: int
+VIDEOWRITER_PROP_NSTRIPES: int
+VIDEOWRITER_PROP_QUALITY: int
+VIDEO_ACCELERATION_ANY: int
+VIDEO_ACCELERATION_D3D11: int
+VIDEO_ACCELERATION_MFX: int
+VIDEO_ACCELERATION_NONE: int
+VIDEO_ACCELERATION_VAAPI: int
+WARP_FILL_OUTLIERS: int
+WARP_INVERSE_MAP: int
+WARP_POLAR_LINEAR: int
+WARP_POLAR_LOG: int
+WINDOW_AUTOSIZE: int
+WINDOW_FREERATIO: int
+WINDOW_FULLSCREEN: int
+WINDOW_GUI_EXPANDED: int
+WINDOW_GUI_NORMAL: int
+WINDOW_KEEPRATIO: int
+WINDOW_NORMAL: int
+WINDOW_OPENGL: int
+WND_PROP_ASPECT_RATIO: int
+WND_PROP_AUTOSIZE: int
+WND_PROP_FULLSCREEN: int
+WND_PROP_OPENGL: int
+WND_PROP_TOPMOST: int
+WND_PROP_VISIBLE: int
+WND_PROP_VSYNC: int
+_INPUT_ARRAY_CUDA_GPU_MAT: int
```

Comment:
Hmm, do we need all these private variables in the stub?

Reply:
I'm not sure whether they're meant to be private or if it's just because the c equivalent is named the same.
But they would be easy to add back if a use-case is found.